### ISSUE TITLE
[Reply] System didn't auto fill email address of reciever when user want to reply an email

### ISSUE THREAD

**Issue by QuangHoang1210** [linagora/tmail-flutter#1466]
https://user-images.githubusercontent.com/124866146/219597711-87dffb3e-2349-448d-afe7-0c55b0082450.mp4

OS: Android, IOS, Window
Device: Mobile/Laptop
TMail version: 0.7.0

*Precondition:
- Inbox existed an email

Step:
1, User login tmail
2, User select an email to read 
3, User tap on relpy icon

Actual: System didn't auto fill email address in "To:" field
Expected: System should auto fill email  address in "To:" field

**Comment by hoangdat** [comment]
maybe field `reply to` is not set for this. please download this email and attach here

**Comment by chibenwa** [comment]
> maybe field reply to is not set for this. please download this email and attach here

If there's no ReplyTo header maybe we could fallback to the From?

Also @QuangHoang1210 how was created the email?

**Comment by QuangHoang1210** [comment]
@chibenwa  @hoangdat  It may happen in case when user using the same email to reply to the same email address.
But it's weirdly. Please following the video bellow: <Compare with "When I click on Reply button and Reply all button">


https://user-images.githubusercontent.com/124866146/219815899-ceea8a2a-8d0b-4ecc-acf7-e207b5aee8da.mp4

**Comment by nqhhdev** [comment]
I downloaded this email. And I saw full information from email. 

Currently, after receiving email information, we are filtering emails to show the `to,cc,bcc` field.
The condition is: Email addresses must be different from the user's email address
I think this is not a bug, it is a user experience. Therefore, we do not support users to autofill their own email when `reply` and `reply to all`. If we are in favor of this then we need to look at this tool in more detail.

cc: @QuangHoang1210
